bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1409,7 +1409,7 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
          ic_ymd="20000101" sim_year="2000" use_excess_ice=".true."
          ic_tod="0" glc_nec="10" use_crop=".true." irrigate=".true."
          phys="clm6_0" use_init_interp=".true."
->lnd/clm2/initdata_esmf/ctsm5.3/ctsm53n04ctsm52028_f09_g17_BgcCrop_exice_hist.clm60.r.2000-01-01.nc
+>lnd/clm2/initdata_esmf/ctsm5.3/ctsm53019_f09_BNF_hist.clm2.r.2000-01-01-00000.nc
```

Comment:
Questions regarding the finidat updates:
1) I updated 1850 and 2000 for f09 only. Should I update ne30 and f19 with the same f09 file or wait until we have spun up ne30 and f19 simulations?
2) For ic_ymd="19790101" should I go back and generate a 1979 file or not worry about it for now (e.g. because we will generate finidat files again before all is said and done)? Right now the closest restart file we have is 1978 from the new simulation.
3) Anything I've forgotten?

Reply:
I'm not sure we'll end up creating ne30 and f19 initial conditions with this file, as Linnia is going to have new suggestions from the PPE soon.  Instead, maybe we point to the f09 initial conditions and set init_interp = true for other resolutions?

Reply:
@wwieder I'm making the change for ne30 to make things easier for coupled cases. It doesn't seem as important for f19 and other resolutions. Let me know if you feel differently.

Reply:
...oh and the main reason I'm leaning against changing for other resolutions is to avoid unnecessarily complicating the namelist_defaults file.

Reply:
It seems like waiting on the 1979 is fine at this juncture. We'll continue to update parameter files and IC files.